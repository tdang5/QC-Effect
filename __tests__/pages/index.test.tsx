import Home from "../../src/pages/index";
import { render } from '@testing-library/react'

describe('Something', () => {
    it('should something', () => {
        const { container } = render(<Home></Home>);
        expect(container).toMatchSnapshot();
    });
});