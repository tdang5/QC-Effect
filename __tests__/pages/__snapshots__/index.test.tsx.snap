// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Something should something 1`] = `
<div>
  <main
    class="css-4ufsix"
  >
    Hello
  </main>
</div>
`;
